lute/std/libs/test.luau

```diff
@@ -213,10 +215,14 @@ function test.run()
 			local fileName, lineNumber = debug.info(4, "sl")
 			local assertName = debug.info(3, "n")
 
+			if assertName == "xpcall" then
```

Comment:
Something I didn't think about before - what happens if your test function itself contains an xpcall that fails. Would we still report the error at the right spot?